pgrx-macros/src/rewriter.rs

```diff
@@ -135,19 +136,52 @@ fn foreign_item_fn(func: &ForeignItemFn, abi: &syn::Abi) -> syn::Result<proc_mac
     let arg_list = rename_arg_list(&func.sig)?;
     let arg_list_with_types = rename_arg_list_with_types(&func.sig)?;
     let return_type = func.sig.output.clone();
+    let link_with_cshim = func.attrs.iter().any(|attr| match &attr.meta {
+        syn::Meta::NameValue(kv) => {
+            if kv.path.get_ident().map(|x| x.to_string()) == Some("link_name".to_string()) {
+                if let syn::Expr::Lit(lit) = &kv.value {
+                    if let syn::Lit::Str(value) = &lit.lit {
```

Comment:
Just do the `syn` imports here so we can make this
```suggestion
                if let Expr::Lit(ExprLit { lit: Lit::Str(value), .. }) = &kv.value {
```